src/components/space-panel/components/visited-nodes/visited-nodes.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react"
+import { CollapsibleList, SimpleListItem } from "@rmwc/list"
+import { Text } from "@netdata/netdata-ui"
+import "@material/list/dist/mdc.list.css"
+import "@rmwc/list/collapsible-list.css"
+import "@rmwc/icon/icon.css"
+import { NodesContainer, ListItem, StyledIcon, ListHeaderContainer, NodeUrl } from "./styled"
+import { agentsMock } from "../../mocks"
+
+const Node = ({ agent: { name, validURLs }, visitNode }: any) => {
+  const urls = validURLs ? Object.keys(validURLs) : []
+  return (
+    <CollapsibleList
+      handle={
+        <SimpleListItem
+          text={
+            <>
+              <StyledIcon name="node" />
+              <Text>{name}</Text>
+            </>
+          }
+          metaIcon={urls.length && "chevron_right"}
+        />
+      }
+    >
+      {urls.map((url: string, i: number) => (
+        <ListItem
+          key={i}
+          onClick={() => {
+            visitNode(url)
+          }}
+        >
+          <NodeUrl>{url}</NodeUrl>
+        </ListItem>
+      ))}
+    </CollapsibleList>
+  )
+}
+
+export const VisitedNodes = () => {
+  const agents = agentsMock as any[] // useSelector(selectAgentsSortedList)
+
+  const visitNode = (url: string) => {
+    if (url.includes("http://") || url.includes("https://")) {
+      window.open(url, "_blank")
+    } else {
+      window.open(`http://${url}`, "_blank")
+    }
+  }
+
+  return (
+    <NodesContainer>
+      <CollapsibleList
+        startOpen
+        handle={
+          <ListHeaderContainer>
```

Comment:
on hover the background has no horizontal padding, shouldn't it cover the whole width (at least on the left)?
![image](https://user-images.githubusercontent.com/5786722/74325513-0d774500-4d89-11ea-940a-f84cfdd2ea4d.png)

Reply:
Maybe it should, however horizontal padding is also used for scrollbar.  This hover comes with rmwc so I have no guidelines for that.

Reply:
Fixed.